PocketKit/Sources/PocketKit/Collections/CollectionMetadataCell.swift

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Kingfisher
+import Textile
+
+class CollectionMetadataCell: UICollectionViewCell {
+    enum Constants {
+        static let stackSpacing: CGFloat = 14
+        static let layoutMargins: UIEdgeInsets = UIEdgeInsets(top: 0, left: 0, bottom: 14, right: 0)
+    }
+
+    struct Model {
+        let byline: NSAttributedString?
+        let itemCount: NSAttributedString?
+        let title: NSAttributedString?
+        let intro: NSAttributedString?
+    }
+
+    private let bylineTextView = ArticleComponentTextView()
+    private let itemCountView = ArticleComponentTextView()
+    private let titleTextView = ArticleComponentTextView()
+    private let introTextView = ArticleComponentTextView()
+
+    private var metaStack: UIStackView = {
```

Comment:
nit: since it's a `UIStackView`, maybe we can tweak the property name to `metaStackView` for clarity? Same for the others